jupyter_server_documents/rooms/yroom_file_api.py

```diff
@@ -95,7 +126,28 @@ class YRoomFileAPI(LoggingConfigurable):
     dual-writes or dirty-reads.
     """
 
+    _last_save_duration: float | None
+    """
+    The duration in seconds of the last save operation. Used to calculate the
+    adaptive poll interval.
+    """
```

Comment:
This instance attribute doesn't seem necessary, since it is only bound to the local variable `save_duration` in the `save()` method.